2022-08-16 02:12:57.854 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:12:57.854 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:12:57.854 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:14:49.816 | ERROR    | grpc_opentracing.grpcext._interceptor:handler:269 - An error has been caught in function 'handler', process 'MainProcess' (28878), thread 'ThreadPoolExecutor-0_0' (123145447325696):
Traceback (most recent call last):

  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 930, in _bootstrap
    self._bootstrap_inner()
    │    └ <function Thread._bootstrap_inner at 0x7fa973ee9c10>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447325696)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 973, in _bootstrap_inner
    self.run()
    │    └ <function Thread.run at 0x7fa973ee9940>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447325696)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 910, in run
    self._target(*self._args, **self._kwargs)
    │    │        │    │        │    └ {}
    │    │        │    │        └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447325696)>
    │    │        │    └ (<weakref at 0x7fa977147e00; to 'ThreadPoolExecutor' at 0x7fa9770dba00>, <_queue.SimpleQueue object at 0x7fa9770f5090>, None,...
    │    │        └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447325696)>
    │    └ <function _worker at 0x7fa9770ea8b0>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447325696)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/concurrent/futures/thread.py", line 77, in _worker
    work_item.run()
    │         └ <function _WorkItem.run at 0x7fa9770ead30>
    └ <concurrent.futures.thread._WorkItem object at 0x7fa977164eb0>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/concurrent/futures/thread.py", line 52, in run
    result = self.fn(*self.args, **self.kwargs)
             │    │   │    │       │    └ {}
             │    │   │    │       └ <concurrent.futures.thread._WorkItem object at 0x7fa977164eb0>
             │    │   │    └ (<grpc._cython.cygrpc.RequestCallEvent object at 0x7fa977153f90>, <grpc._server._RPCState object at 0x7fa977164430>, <functio...
             │    │   └ <concurrent.futures.thread._WorkItem object at 0x7fa977164eb0>
             │    └ <function _unary_response_in_pool at 0x7fa97712bd30>
             └ <concurrent.futures.thread._WorkItem object at 0x7fa977164eb0>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/grpc/_server.py", line 560, in _unary_response_in_pool
    response, proceed = _call_behavior(rpc_event, state, behavior,
                        │              │          │      └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation at 0x7fa9771468b0>
                        │              │          └ <grpc._server._RPCState object at 0x7fa977164430>
                        │              └ <grpc._cython.cygrpc.RequestCallEvent object at 0x7fa977153f90>
                        └ <function _call_behavior at 0x7fa97712b940>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/grpc/_server.py", line 443, in _call_behavior
    response_or_iterator = behavior(argument, context)
                           │        │         └ <grpc._server._Context object at 0x7fa977164f70>
                           │        └ user_id: 1
                           │          name: "\351\231\210\345\213\2072"
                           │          mobile: "18612917508"
                           │          address: "\345\215\216\344\270\255"
                           │          post: "Kyrykjijk Ptkev...
                           └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation at 0x7fa9771468b0>

  File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/grpcext/_interceptor.py", line 272, in adaptation
    return self._interceptor.intercept_unary(
           │    │            └ <function OpenTracingServerInterceptor.intercept_unary at 0x7fa9770fb9d0>
           │    └ <grpc_opentracing._server.OpenTracingServerInterceptor object at 0x7fa9770dbfa0>
           └ <grpc_opentracing.grpcext._interceptor._InterceptorRpcMethodHandler object at 0x7fa977164130>

  File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/_server.py", line 154, in intercept_unary
    response = handler(request, servicer_context)
               │       │        └ <grpc_opentracing._server._OpenTracingServicerContext object at 0x7fa977164c70>
               │       └ user_id: 1
               │         name: "\351\231\210\345\213\2072"
               │         mobile: "18612917508"
               │         address: "\345\215\216\344\270\255"
               │         post: "Kyrykjijk Ptkev...
               └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation.<locals>.handler at 0x7fa977146940>

> File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/grpcext/_interceptor.py", line 269, in handler
    return self._rpc_method_handler.unary_unary(request,
           │    │                   │           └ user_id: 1
           │    │                   │             name: "\351\231\210\345\213\2072"
           │    │                   │             mobile: "18612917508"
           │    │                   │             address: "\345\215\216\344\270\255"
           │    │                   │             post: "Kyrykjijk Ptkev...
           │    │                   └ _tuplegetter(4, 'Alias for field number 4')
           │    └ RpcMethodHandler(request_streaming=False, response_streaming=False, request_deserializer=<built-in method FromString of Gener...
           └ <grpc_opentracing.grpcext._interceptor._InterceptorRpcMethodHandler object at 0x7fa977164130>

  File "/Users/www/learn-note/gpmall/daoServer/order/handler/order.py", line 156, in CreateOrder
    ret = p.send_message_in_transaction(msg, self.local_execute, user_args=None)
          │ │                           │    │    └ <function OrderServicer.local_execute at 0x7fa976559940>
          │ │                           │    └ <handler.order.OrderServicer object at 0x7fa9771218e0>
          │ │                           └ <rocketmq.client.Message object at 0x7fa977164b80>
          │ └ <function TransactionMQProducer.send_message_in_transaction at 0x7fa975c4fa60>
          └ <rocketmq.client.TransactionMQProducer object at 0x7fa977164cd0>

  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/rocketmq/client.py", line 335, in send_message_in_transaction
    ffi_check(
    └ <function ffi_check at 0x7fa975c379d0>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/rocketmq/exceptions.py", line 44, in ffi_check
    raise exc_cls(msg)
          │       └ 'msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> li...
          └ <class 'rocketmq.exceptions.RocketMQException'>

rocketmq.exceptions.RocketMQException: msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53
2022-08-16 02:14:49.816 | ERROR    | grpc_opentracing.grpcext._interceptor:handler:269 - An error has been caught in function 'handler', process 'MainProcess' (28878), thread 'ThreadPoolExecutor-0_0' (123145447325696):
Traceback (most recent call last):

  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 930, in _bootstrap
    self._bootstrap_inner()
    │    └ <function Thread._bootstrap_inner at 0x7fa973ee9c10>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447325696)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 973, in _bootstrap_inner
    self.run()
    │    └ <function Thread.run at 0x7fa973ee9940>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447325696)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 910, in run
    self._target(*self._args, **self._kwargs)
    │    │        │    │        │    └ {}
    │    │        │    │        └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447325696)>
    │    │        │    └ (<weakref at 0x7fa977147e00; to 'ThreadPoolExecutor' at 0x7fa9770dba00>, <_queue.SimpleQueue object at 0x7fa9770f5090>, None,...
    │    │        └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447325696)>
    │    └ <function _worker at 0x7fa9770ea8b0>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447325696)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/concurrent/futures/thread.py", line 77, in _worker
    work_item.run()
    │         └ <function _WorkItem.run at 0x7fa9770ead30>
    └ <concurrent.futures.thread._WorkItem object at 0x7fa977164eb0>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/concurrent/futures/thread.py", line 52, in run
    result = self.fn(*self.args, **self.kwargs)
             │    │   │    │       │    └ {}
             │    │   │    │       └ <concurrent.futures.thread._WorkItem object at 0x7fa977164eb0>
             │    │   │    └ (<grpc._cython.cygrpc.RequestCallEvent object at 0x7fa977153f90>, <grpc._server._RPCState object at 0x7fa977164430>, <functio...
             │    │   └ <concurrent.futures.thread._WorkItem object at 0x7fa977164eb0>
             │    └ <function _unary_response_in_pool at 0x7fa97712bd30>
             └ <concurrent.futures.thread._WorkItem object at 0x7fa977164eb0>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/grpc/_server.py", line 560, in _unary_response_in_pool
    response, proceed = _call_behavior(rpc_event, state, behavior,
                        │              │          │      └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation at 0x7fa9771468b0>
                        │              │          └ <grpc._server._RPCState object at 0x7fa977164430>
                        │              └ <grpc._cython.cygrpc.RequestCallEvent object at 0x7fa977153f90>
                        └ <function _call_behavior at 0x7fa97712b940>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/grpc/_server.py", line 443, in _call_behavior
    response_or_iterator = behavior(argument, context)
                           │        │         └ <grpc._server._Context object at 0x7fa977164f70>
                           │        └ user_id: 1
                           │          name: "\351\231\210\345\213\2072"
                           │          mobile: "18612917508"
                           │          address: "\345\215\216\344\270\255"
                           │          post: "Kyrykjijk Ptkev...
                           └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation at 0x7fa9771468b0>

  File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/grpcext/_interceptor.py", line 272, in adaptation
    return self._interceptor.intercept_unary(
           │    │            └ <function OpenTracingServerInterceptor.intercept_unary at 0x7fa9770fb9d0>
           │    └ <grpc_opentracing._server.OpenTracingServerInterceptor object at 0x7fa9770dbfa0>
           └ <grpc_opentracing.grpcext._interceptor._InterceptorRpcMethodHandler object at 0x7fa977164130>

  File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/_server.py", line 154, in intercept_unary
    response = handler(request, servicer_context)
               │       │        └ <grpc_opentracing._server._OpenTracingServicerContext object at 0x7fa977164c70>
               │       └ user_id: 1
               │         name: "\351\231\210\345\213\2072"
               │         mobile: "18612917508"
               │         address: "\345\215\216\344\270\255"
               │         post: "Kyrykjijk Ptkev...
               └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation.<locals>.handler at 0x7fa977146940>

> File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/grpcext/_interceptor.py", line 269, in handler
    return self._rpc_method_handler.unary_unary(request,
           │    │                   │           └ user_id: 1
           │    │                   │             name: "\351\231\210\345\213\2072"
           │    │                   │             mobile: "18612917508"
           │    │                   │             address: "\345\215\216\344\270\255"
           │    │                   │             post: "Kyrykjijk Ptkev...
           │    │                   └ _tuplegetter(4, 'Alias for field number 4')
           │    └ RpcMethodHandler(request_streaming=False, response_streaming=False, request_deserializer=<built-in method FromString of Gener...
           └ <grpc_opentracing.grpcext._interceptor._InterceptorRpcMethodHandler object at 0x7fa977164130>

  File "/Users/www/learn-note/gpmall/daoServer/order/handler/order.py", line 156, in CreateOrder
    ret = p.send_message_in_transaction(msg, self.local_execute, user_args=None)
          │ │                           │    │    └ <function OrderServicer.local_execute at 0x7fa976559940>
          │ │                           │    └ <handler.order.OrderServicer object at 0x7fa9771218e0>
          │ │                           └ <rocketmq.client.Message object at 0x7fa977164b80>
          │ └ <function TransactionMQProducer.send_message_in_transaction at 0x7fa975c4fa60>
          └ <rocketmq.client.TransactionMQProducer object at 0x7fa977164cd0>

  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/rocketmq/client.py", line 335, in send_message_in_transaction
    ffi_check(
    └ <function ffi_check at 0x7fa975c379d0>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/rocketmq/exceptions.py", line 44, in ffi_check
    raise exc_cls(msg)
          │       └ 'msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> li...
          └ <class 'rocketmq.exceptions.RocketMQException'>

rocketmq.exceptions.RocketMQException: msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53
2022-08-16 02:14:49.816 | ERROR    | grpc_opentracing.grpcext._interceptor:handler:269 - An error has been caught in function 'handler', process 'MainProcess' (28878), thread 'ThreadPoolExecutor-0_0' (123145447325696):
Traceback (most recent call last):

  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 930, in _bootstrap
    self._bootstrap_inner()
    │    └ <function Thread._bootstrap_inner at 0x7fa973ee9c10>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447325696)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 973, in _bootstrap_inner
    self.run()
    │    └ <function Thread.run at 0x7fa973ee9940>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447325696)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 910, in run
    self._target(*self._args, **self._kwargs)
    │    │        │    │        │    └ {}
    │    │        │    │        └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447325696)>
    │    │        │    └ (<weakref at 0x7fa977147e00; to 'ThreadPoolExecutor' at 0x7fa9770dba00>, <_queue.SimpleQueue object at 0x7fa9770f5090>, None,...
    │    │        └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447325696)>
    │    └ <function _worker at 0x7fa9770ea8b0>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447325696)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/concurrent/futures/thread.py", line 77, in _worker
    work_item.run()
    │         └ <function _WorkItem.run at 0x7fa9770ead30>
    └ <concurrent.futures.thread._WorkItem object at 0x7fa977164eb0>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/concurrent/futures/thread.py", line 52, in run
    result = self.fn(*self.args, **self.kwargs)
             │    │   │    │       │    └ {}
             │    │   │    │       └ <concurrent.futures.thread._WorkItem object at 0x7fa977164eb0>
             │    │   │    └ (<grpc._cython.cygrpc.RequestCallEvent object at 0x7fa977153f90>, <grpc._server._RPCState object at 0x7fa977164430>, <functio...
             │    │   └ <concurrent.futures.thread._WorkItem object at 0x7fa977164eb0>
             │    └ <function _unary_response_in_pool at 0x7fa97712bd30>
             └ <concurrent.futures.thread._WorkItem object at 0x7fa977164eb0>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/grpc/_server.py", line 560, in _unary_response_in_pool
    response, proceed = _call_behavior(rpc_event, state, behavior,
                        │              │          │      └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation at 0x7fa9771468b0>
                        │              │          └ <grpc._server._RPCState object at 0x7fa977164430>
                        │              └ <grpc._cython.cygrpc.RequestCallEvent object at 0x7fa977153f90>
                        └ <function _call_behavior at 0x7fa97712b940>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/grpc/_server.py", line 443, in _call_behavior
    response_or_iterator = behavior(argument, context)
                           │        │         └ <grpc._server._Context object at 0x7fa977164f70>
                           │        └ user_id: 1
                           │          name: "\351\231\210\345\213\2072"
                           │          mobile: "18612917508"
                           │          address: "\345\215\216\344\270\255"
                           │          post: "Kyrykjijk Ptkev...
                           └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation at 0x7fa9771468b0>

  File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/grpcext/_interceptor.py", line 272, in adaptation
    return self._interceptor.intercept_unary(
           │    │            └ <function OpenTracingServerInterceptor.intercept_unary at 0x7fa9770fb9d0>
           │    └ <grpc_opentracing._server.OpenTracingServerInterceptor object at 0x7fa9770dbfa0>
           └ <grpc_opentracing.grpcext._interceptor._InterceptorRpcMethodHandler object at 0x7fa977164130>

  File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/_server.py", line 154, in intercept_unary
    response = handler(request, servicer_context)
               │       │        └ <grpc_opentracing._server._OpenTracingServicerContext object at 0x7fa977164c70>
               │       └ user_id: 1
               │         name: "\351\231\210\345\213\2072"
               │         mobile: "18612917508"
               │         address: "\345\215\216\344\270\255"
               │         post: "Kyrykjijk Ptkev...
               └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation.<locals>.handler at 0x7fa977146940>

> File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/grpcext/_interceptor.py", line 269, in handler
    return self._rpc_method_handler.unary_unary(request,
           │    │                   │           └ user_id: 1
           │    │                   │             name: "\351\231\210\345\213\2072"
           │    │                   │             mobile: "18612917508"
           │    │                   │             address: "\345\215\216\344\270\255"
           │    │                   │             post: "Kyrykjijk Ptkev...
           │    │                   └ _tuplegetter(4, 'Alias for field number 4')
           │    └ RpcMethodHandler(request_streaming=False, response_streaming=False, request_deserializer=<built-in method FromString of Gener...
           └ <grpc_opentracing.grpcext._interceptor._InterceptorRpcMethodHandler object at 0x7fa977164130>

  File "/Users/www/learn-note/gpmall/daoServer/order/handler/order.py", line 156, in CreateOrder
    ret = p.send_message_in_transaction(msg, self.local_execute, user_args=None)
          │ │                           │    │    └ <function OrderServicer.local_execute at 0x7fa976559940>
          │ │                           │    └ <handler.order.OrderServicer object at 0x7fa9771218e0>
          │ │                           └ <rocketmq.client.Message object at 0x7fa977164b80>
          │ └ <function TransactionMQProducer.send_message_in_transaction at 0x7fa975c4fa60>
          └ <rocketmq.client.TransactionMQProducer object at 0x7fa977164cd0>

  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/rocketmq/client.py", line 335, in send_message_in_transaction
    ffi_check(
    └ <function ffi_check at 0x7fa975c379d0>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/rocketmq/exceptions.py", line 44, in ffi_check
    raise exc_cls(msg)
          │       └ 'msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> li...
          └ <class 'rocketmq.exceptions.RocketMQException'>

rocketmq.exceptions.RocketMQException: msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53
2022-08-16 02:16:36.573 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:16:36.573 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:16:36.573 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:16:42.817 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:16:42.817 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:16:42.817 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:17:11.313 | ERROR    | grpc_opentracing.grpcext._interceptor:handler:269 - An error has been caught in function 'handler', process 'MainProcess' (29129), thread 'ThreadPoolExecutor-0_0' (123145447309312):
Traceback (most recent call last):

  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 930, in _bootstrap
    self._bootstrap_inner()
    │    └ <function Thread._bootstrap_inner at 0x7fb164ee9c10>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447309312)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 973, in _bootstrap_inner
    self.run()
    │    └ <function Thread.run at 0x7fb164ee9940>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447309312)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 910, in run
    self._target(*self._args, **self._kwargs)
    │    │        │    │        │    └ {}
    │    │        │    │        └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447309312)>
    │    │        │    └ (<weakref at 0x7fb168986f90; to 'ThreadPoolExecutor' at 0x7fb168931040>, <_queue.SimpleQueue object at 0x7fb168934f90>, None,...
    │    │        └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447309312)>
    │    └ <function _worker at 0x7fb16892b8b0>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447309312)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/concurrent/futures/thread.py", line 77, in _worker
    work_item.run()
    │         └ <function _WorkItem.run at 0x7fb16892bd30>
    └ <concurrent.futures.thread._WorkItem object at 0x7fb16898eb80>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/concurrent/futures/thread.py", line 52, in run
    result = self.fn(*self.args, **self.kwargs)
             │    │   │    │       │    └ {}
             │    │   │    │       └ <concurrent.futures.thread._WorkItem object at 0x7fb16898eb80>
             │    │   │    └ (<grpc._cython.cygrpc.RequestCallEvent object at 0x7fb168986bd0>, <grpc._server._RPCState object at 0x7fb16891b820>, <functio...
             │    │   └ <concurrent.futures.thread._WorkItem object at 0x7fb16898eb80>
             │    └ <function _unary_response_in_pool at 0x7fb16896cf70>
             └ <concurrent.futures.thread._WorkItem object at 0x7fb16898eb80>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/grpc/_server.py", line 560, in _unary_response_in_pool
    response, proceed = _call_behavior(rpc_event, state, behavior,
                        │              │          │      └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation at 0x7fb168985e50>
                        │              │          └ <grpc._server._RPCState object at 0x7fb16891b820>
                        │              └ <grpc._cython.cygrpc.RequestCallEvent object at 0x7fb168986bd0>
                        └ <function _call_behavior at 0x7fb16896cb80>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/grpc/_server.py", line 443, in _call_behavior
    response_or_iterator = behavior(argument, context)
                           │        │         └ <grpc._server._Context object at 0x7fb1689a80d0>
                           │        └ user_id: 1
                           │          name: "\351\203\221\347\247\200\350\213\261"
                           │          mobile: "18612917508"
                           │          address: "\344\270\234\345\214\227"
                           │          post: "Kfmy...
                           └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation at 0x7fb168985e50>

  File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/grpcext/_interceptor.py", line 272, in adaptation
    return self._interceptor.intercept_unary(
           │    │            └ <function OpenTracingServerInterceptor.intercept_unary at 0x7fb16893b9d0>
           │    └ <grpc_opentracing._server.OpenTracingServerInterceptor object at 0x7fb16891bfa0>
           └ <grpc_opentracing.grpcext._interceptor._InterceptorRpcMethodHandler object at 0x7fb16898e5e0>

  File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/_server.py", line 154, in intercept_unary
    response = handler(request, servicer_context)
               │       │        └ <grpc_opentracing._server._OpenTracingServicerContext object at 0x7fb1689a82b0>
               │       └ user_id: 1
               │         name: "\351\203\221\347\247\200\350\213\261"
               │         mobile: "18612917508"
               │         address: "\344\270\234\345\214\227"
               │         post: "Kfmy...
               └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation.<locals>.handler at 0x7fb1689a90d0>

> File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/grpcext/_interceptor.py", line 269, in handler
    return self._rpc_method_handler.unary_unary(request,
           │    │                   │           └ user_id: 1
           │    │                   │             name: "\351\203\221\347\247\200\350\213\261"
           │    │                   │             mobile: "18612917508"
           │    │                   │             address: "\344\270\234\345\214\227"
           │    │                   │             post: "Kfmy...
           │    │                   └ _tuplegetter(4, 'Alias for field number 4')
           │    └ RpcMethodHandler(request_streaming=False, response_streaming=False, request_deserializer=<built-in method FromString of Gener...
           └ <grpc_opentracing.grpcext._interceptor._InterceptorRpcMethodHandler object at 0x7fb16898e5e0>

  File "/Users/www/learn-note/gpmall/daoServer/order/handler/order.py", line 156, in CreateOrder
    ret = p.send_message_in_transaction(msg, self.local_execute, user_args=None)
          │ │                           │    │    └ <function OrderServicer.local_execute at 0x7fb167c58940>
          │ │                           │    └ <handler.order.OrderServicer object at 0x7fb168961ca0>
          │ │                           └ <rocketmq.client.Message object at 0x7fb1689a83a0>
          │ └ <function TransactionMQProducer.send_message_in_transaction at 0x7fb166b4fa60>
          └ <rocketmq.client.TransactionMQProducer object at 0x7fb1689a8310>

  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/rocketmq/client.py", line 335, in send_message_in_transaction
    ffi_check(
    └ <function ffi_check at 0x7fb166b379d0>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/rocketmq/exceptions.py", line 44, in ffi_check
    raise exc_cls(msg)
          │       └ 'msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> li...
          └ <class 'rocketmq.exceptions.RocketMQException'>

rocketmq.exceptions.RocketMQException: msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53
2022-08-16 02:17:11.313 | ERROR    | grpc_opentracing.grpcext._interceptor:handler:269 - An error has been caught in function 'handler', process 'MainProcess' (29129), thread 'ThreadPoolExecutor-0_0' (123145447309312):
Traceback (most recent call last):

  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 930, in _bootstrap
    self._bootstrap_inner()
    │    └ <function Thread._bootstrap_inner at 0x7fb164ee9c10>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447309312)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 973, in _bootstrap_inner
    self.run()
    │    └ <function Thread.run at 0x7fb164ee9940>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447309312)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 910, in run
    self._target(*self._args, **self._kwargs)
    │    │        │    │        │    └ {}
    │    │        │    │        └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447309312)>
    │    │        │    └ (<weakref at 0x7fb168986f90; to 'ThreadPoolExecutor' at 0x7fb168931040>, <_queue.SimpleQueue object at 0x7fb168934f90>, None,...
    │    │        └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447309312)>
    │    └ <function _worker at 0x7fb16892b8b0>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447309312)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/concurrent/futures/thread.py", line 77, in _worker
    work_item.run()
    │         └ <function _WorkItem.run at 0x7fb16892bd30>
    └ <concurrent.futures.thread._WorkItem object at 0x7fb16898eb80>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/concurrent/futures/thread.py", line 52, in run
    result = self.fn(*self.args, **self.kwargs)
             │    │   │    │       │    └ {}
             │    │   │    │       └ <concurrent.futures.thread._WorkItem object at 0x7fb16898eb80>
             │    │   │    └ (<grpc._cython.cygrpc.RequestCallEvent object at 0x7fb168986bd0>, <grpc._server._RPCState object at 0x7fb16891b820>, <functio...
             │    │   └ <concurrent.futures.thread._WorkItem object at 0x7fb16898eb80>
             │    └ <function _unary_response_in_pool at 0x7fb16896cf70>
             └ <concurrent.futures.thread._WorkItem object at 0x7fb16898eb80>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/grpc/_server.py", line 560, in _unary_response_in_pool
    response, proceed = _call_behavior(rpc_event, state, behavior,
                        │              │          │      └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation at 0x7fb168985e50>
                        │              │          └ <grpc._server._RPCState object at 0x7fb16891b820>
                        │              └ <grpc._cython.cygrpc.RequestCallEvent object at 0x7fb168986bd0>
                        └ <function _call_behavior at 0x7fb16896cb80>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/grpc/_server.py", line 443, in _call_behavior
    response_or_iterator = behavior(argument, context)
                           │        │         └ <grpc._server._Context object at 0x7fb1689a80d0>
                           │        └ user_id: 1
                           │          name: "\351\203\221\347\247\200\350\213\261"
                           │          mobile: "18612917508"
                           │          address: "\344\270\234\345\214\227"
                           │          post: "Kfmy...
                           └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation at 0x7fb168985e50>

  File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/grpcext/_interceptor.py", line 272, in adaptation
    return self._interceptor.intercept_unary(
           │    │            └ <function OpenTracingServerInterceptor.intercept_unary at 0x7fb16893b9d0>
           │    └ <grpc_opentracing._server.OpenTracingServerInterceptor object at 0x7fb16891bfa0>
           └ <grpc_opentracing.grpcext._interceptor._InterceptorRpcMethodHandler object at 0x7fb16898e5e0>

  File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/_server.py", line 154, in intercept_unary
    response = handler(request, servicer_context)
               │       │        └ <grpc_opentracing._server._OpenTracingServicerContext object at 0x7fb1689a82b0>
               │       └ user_id: 1
               │         name: "\351\203\221\347\247\200\350\213\261"
               │         mobile: "18612917508"
               │         address: "\344\270\234\345\214\227"
               │         post: "Kfmy...
               └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation.<locals>.handler at 0x7fb1689a90d0>

> File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/grpcext/_interceptor.py", line 269, in handler
    return self._rpc_method_handler.unary_unary(request,
           │    │                   │           └ user_id: 1
           │    │                   │             name: "\351\203\221\347\247\200\350\213\261"
           │    │                   │             mobile: "18612917508"
           │    │                   │             address: "\344\270\234\345\214\227"
           │    │                   │             post: "Kfmy...
           │    │                   └ _tuplegetter(4, 'Alias for field number 4')
           │    └ RpcMethodHandler(request_streaming=False, response_streaming=False, request_deserializer=<built-in method FromString of Gener...
           └ <grpc_opentracing.grpcext._interceptor._InterceptorRpcMethodHandler object at 0x7fb16898e5e0>

  File "/Users/www/learn-note/gpmall/daoServer/order/handler/order.py", line 156, in CreateOrder
    ret = p.send_message_in_transaction(msg, self.local_execute, user_args=None)
          │ │                           │    │    └ <function OrderServicer.local_execute at 0x7fb167c58940>
          │ │                           │    └ <handler.order.OrderServicer object at 0x7fb168961ca0>
          │ │                           └ <rocketmq.client.Message object at 0x7fb1689a83a0>
          │ └ <function TransactionMQProducer.send_message_in_transaction at 0x7fb166b4fa60>
          └ <rocketmq.client.TransactionMQProducer object at 0x7fb1689a8310>

  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/rocketmq/client.py", line 335, in send_message_in_transaction
    ffi_check(
    └ <function ffi_check at 0x7fb166b379d0>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/rocketmq/exceptions.py", line 44, in ffi_check
    raise exc_cls(msg)
          │       └ 'msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> li...
          └ <class 'rocketmq.exceptions.RocketMQException'>

rocketmq.exceptions.RocketMQException: msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53
2022-08-16 02:17:11.313 | ERROR    | grpc_opentracing.grpcext._interceptor:handler:269 - An error has been caught in function 'handler', process 'MainProcess' (29129), thread 'ThreadPoolExecutor-0_0' (123145447309312):
Traceback (most recent call last):

  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 930, in _bootstrap
    self._bootstrap_inner()
    │    └ <function Thread._bootstrap_inner at 0x7fb164ee9c10>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447309312)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 973, in _bootstrap_inner
    self.run()
    │    └ <function Thread.run at 0x7fb164ee9940>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447309312)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 910, in run
    self._target(*self._args, **self._kwargs)
    │    │        │    │        │    └ {}
    │    │        │    │        └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447309312)>
    │    │        │    └ (<weakref at 0x7fb168986f90; to 'ThreadPoolExecutor' at 0x7fb168931040>, <_queue.SimpleQueue object at 0x7fb168934f90>, None,...
    │    │        └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447309312)>
    │    └ <function _worker at 0x7fb16892b8b0>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145447309312)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/concurrent/futures/thread.py", line 77, in _worker
    work_item.run()
    │         └ <function _WorkItem.run at 0x7fb16892bd30>
    └ <concurrent.futures.thread._WorkItem object at 0x7fb16898eb80>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/concurrent/futures/thread.py", line 52, in run
    result = self.fn(*self.args, **self.kwargs)
             │    │   │    │       │    └ {}
             │    │   │    │       └ <concurrent.futures.thread._WorkItem object at 0x7fb16898eb80>
             │    │   │    └ (<grpc._cython.cygrpc.RequestCallEvent object at 0x7fb168986bd0>, <grpc._server._RPCState object at 0x7fb16891b820>, <functio...
             │    │   └ <concurrent.futures.thread._WorkItem object at 0x7fb16898eb80>
             │    └ <function _unary_response_in_pool at 0x7fb16896cf70>
             └ <concurrent.futures.thread._WorkItem object at 0x7fb16898eb80>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/grpc/_server.py", line 560, in _unary_response_in_pool
    response, proceed = _call_behavior(rpc_event, state, behavior,
                        │              │          │      └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation at 0x7fb168985e50>
                        │              │          └ <grpc._server._RPCState object at 0x7fb16891b820>
                        │              └ <grpc._cython.cygrpc.RequestCallEvent object at 0x7fb168986bd0>
                        └ <function _call_behavior at 0x7fb16896cb80>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/grpc/_server.py", line 443, in _call_behavior
    response_or_iterator = behavior(argument, context)
                           │        │         └ <grpc._server._Context object at 0x7fb1689a80d0>
                           │        └ user_id: 1
                           │          name: "\351\203\221\347\247\200\350\213\261"
                           │          mobile: "18612917508"
                           │          address: "\344\270\234\345\214\227"
                           │          post: "Kfmy...
                           └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation at 0x7fb168985e50>

  File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/grpcext/_interceptor.py", line 272, in adaptation
    return self._interceptor.intercept_unary(
           │    │            └ <function OpenTracingServerInterceptor.intercept_unary at 0x7fb16893b9d0>
           │    └ <grpc_opentracing._server.OpenTracingServerInterceptor object at 0x7fb16891bfa0>
           └ <grpc_opentracing.grpcext._interceptor._InterceptorRpcMethodHandler object at 0x7fb16898e5e0>

  File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/_server.py", line 154, in intercept_unary
    response = handler(request, servicer_context)
               │       │        └ <grpc_opentracing._server._OpenTracingServicerContext object at 0x7fb1689a82b0>
               │       └ user_id: 1
               │         name: "\351\203\221\347\247\200\350\213\261"
               │         mobile: "18612917508"
               │         address: "\344\270\234\345\214\227"
               │         post: "Kfmy...
               └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation.<locals>.handler at 0x7fb1689a90d0>

> File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/grpcext/_interceptor.py", line 269, in handler
    return self._rpc_method_handler.unary_unary(request,
           │    │                   │           └ user_id: 1
           │    │                   │             name: "\351\203\221\347\247\200\350\213\261"
           │    │                   │             mobile: "18612917508"
           │    │                   │             address: "\344\270\234\345\214\227"
           │    │                   │             post: "Kfmy...
           │    │                   └ _tuplegetter(4, 'Alias for field number 4')
           │    └ RpcMethodHandler(request_streaming=False, response_streaming=False, request_deserializer=<built-in method FromString of Gener...
           └ <grpc_opentracing.grpcext._interceptor._InterceptorRpcMethodHandler object at 0x7fb16898e5e0>

  File "/Users/www/learn-note/gpmall/daoServer/order/handler/order.py", line 156, in CreateOrder
    ret = p.send_message_in_transaction(msg, self.local_execute, user_args=None)
          │ │                           │    │    └ <function OrderServicer.local_execute at 0x7fb167c58940>
          │ │                           │    └ <handler.order.OrderServicer object at 0x7fb168961ca0>
          │ │                           └ <rocketmq.client.Message object at 0x7fb1689a83a0>
          │ └ <function TransactionMQProducer.send_message_in_transaction at 0x7fb166b4fa60>
          └ <rocketmq.client.TransactionMQProducer object at 0x7fb1689a8310>

  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/rocketmq/client.py", line 335, in send_message_in_transaction
    ffi_check(
    └ <function ffi_check at 0x7fb166b379d0>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/rocketmq/exceptions.py", line 44, in ffi_check
    raise exc_cls(msg)
          │       └ 'msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> li...
          └ <class 'rocketmq.exceptions.RocketMQException'>

rocketmq.exceptions.RocketMQException: msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53
2022-08-16 02:20:17.598 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:20:17.598 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:20:17.598 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:20:22.049 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:20:22.049 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:20:22.049 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:20:32.956 | ERROR    | grpc_opentracing.grpcext._interceptor:handler:269 - An error has been caught in function 'handler', process 'MainProcess' (29350), thread 'ThreadPoolExecutor-0_0' (123145521827840):
Traceback (most recent call last):

  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 930, in _bootstrap
    self._bootstrap_inner()
    │    └ <function Thread._bootstrap_inner at 0x7fee736e9c10>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145521827840)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 973, in _bootstrap_inner
    self.run()
    │    └ <function Thread.run at 0x7fee736e9940>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145521827840)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 910, in run
    self._target(*self._args, **self._kwargs)
    │    │        │    │        │    └ {}
    │    │        │    │        └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145521827840)>
    │    │        │    └ (<weakref at 0x7fee76941e50; to 'ThreadPoolExecutor' at 0x7fee75f9dfa0>, <_queue.SimpleQueue object at 0x7fee75fb7450>, None,...
    │    │        └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145521827840)>
    │    └ <function _worker at 0x7fee75fab8b0>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145521827840)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/concurrent/futures/thread.py", line 77, in _worker
    work_item.run()
    │         └ <function _WorkItem.run at 0x7fee75fabd30>
    └ <concurrent.futures.thread._WorkItem object at 0x7fee7694f970>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/concurrent/futures/thread.py", line 52, in run
    result = self.fn(*self.args, **self.kwargs)
             │    │   │    │       │    └ {}
             │    │   │    │       └ <concurrent.futures.thread._WorkItem object at 0x7fee7694f970>
             │    │   │    └ (<grpc._cython.cygrpc.RequestCallEvent object at 0x7fee769419a0>, <grpc._server._RPCState object at 0x7fee769427f0>, <functio...
             │    │   └ <concurrent.futures.thread._WorkItem object at 0x7fee7694f970>
             │    └ <function _unary_response_in_pool at 0x7fee7692bd30>
             └ <concurrent.futures.thread._WorkItem object at 0x7fee7694f970>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/grpc/_server.py", line 560, in _unary_response_in_pool
    response, proceed = _call_behavior(rpc_event, state, behavior,
                        │              │          │      └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation at 0x7fee76946ca0>
                        │              │          └ <grpc._server._RPCState object at 0x7fee769427f0>
                        │              └ <grpc._cython.cygrpc.RequestCallEvent object at 0x7fee769419a0>
                        └ <function _call_behavior at 0x7fee7692b940>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/grpc/_server.py", line 443, in _call_behavior
    response_or_iterator = behavior(argument, context)
                           │        │         └ <grpc._server._Context object at 0x7fee7694fa00>
                           │        └ user_id: 1
                           │          name: "\346\256\265\350\266\205123"
                           │          mobile: "18612917508"
                           │          address: "\345\215\216\344\270\2551code"
                           │          
                           └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation at 0x7fee76946ca0>

  File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/grpcext/_interceptor.py", line 272, in adaptation
    return self._interceptor.intercept_unary(
           │    │            └ <function OpenTracingServerInterceptor.intercept_unary at 0x7fee75fbd9d0>
           │    └ <grpc_opentracing._server.OpenTracingServerInterceptor object at 0x7fee75f9df40>
           └ <grpc_opentracing.grpcext._interceptor._InterceptorRpcMethodHandler object at 0x7fee769427c0>

  File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/_server.py", line 154, in intercept_unary
    response = handler(request, servicer_context)
               │       │        └ <grpc_opentracing._server._OpenTracingServicerContext object at 0x7fee7694fc70>
               │       └ user_id: 1
               │         name: "\346\256\265\350\266\205123"
               │         mobile: "18612917508"
               │         address: "\345\215\216\344\270\2551code"
               │         
               └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation.<locals>.handler at 0x7fee76946940>

> File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/grpcext/_interceptor.py", line 269, in handler
    return self._rpc_method_handler.unary_unary(request,
           │    │                   │           └ user_id: 1
           │    │                   │             name: "\346\256\265\350\266\205123"
           │    │                   │             mobile: "18612917508"
           │    │                   │             address: "\345\215\216\344\270\2551code"
           │    │                   │             
           │    │                   └ _tuplegetter(4, 'Alias for field number 4')
           │    └ RpcMethodHandler(request_streaming=False, response_streaming=False, request_deserializer=<built-in method FromString of Gener...
           └ <grpc_opentracing.grpcext._interceptor._InterceptorRpcMethodHandler object at 0x7fee769427c0>

  File "/Users/www/learn-note/gpmall/daoServer/order/handler/order.py", line 156, in CreateOrder
    ret = p.send_message_in_transaction(msg, self.local_execute, user_args=None)
          │ │                           │    │    └ <function OrderServicer.local_execute at 0x7fee75c48940>
          │ │                           │    └ <handler.order.OrderServicer object at 0x7fee76921ac0>
          │ │                           └ <rocketmq.client.Message object at 0x7fee7694fd60>
          │ └ <function TransactionMQProducer.send_message_in_transaction at 0x7fee7544fa60>
          └ <rocketmq.client.TransactionMQProducer object at 0x7fee7694fcd0>

  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/rocketmq/client.py", line 335, in send_message_in_transaction
    ffi_check(
    └ <function ffi_check at 0x7fee754379d0>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/rocketmq/exceptions.py", line 44, in ffi_check
    raise exc_cls(msg)
          │       └ 'msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> li...
          └ <class 'rocketmq.exceptions.RocketMQException'>

rocketmq.exceptions.RocketMQException: msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53
2022-08-16 02:20:32.956 | ERROR    | grpc_opentracing.grpcext._interceptor:handler:269 - An error has been caught in function 'handler', process 'MainProcess' (29350), thread 'ThreadPoolExecutor-0_0' (123145521827840):
Traceback (most recent call last):

  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 930, in _bootstrap
    self._bootstrap_inner()
    │    └ <function Thread._bootstrap_inner at 0x7fee736e9c10>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145521827840)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 973, in _bootstrap_inner
    self.run()
    │    └ <function Thread.run at 0x7fee736e9940>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145521827840)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 910, in run
    self._target(*self._args, **self._kwargs)
    │    │        │    │        │    └ {}
    │    │        │    │        └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145521827840)>
    │    │        │    └ (<weakref at 0x7fee76941e50; to 'ThreadPoolExecutor' at 0x7fee75f9dfa0>, <_queue.SimpleQueue object at 0x7fee75fb7450>, None,...
    │    │        └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145521827840)>
    │    └ <function _worker at 0x7fee75fab8b0>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145521827840)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/concurrent/futures/thread.py", line 77, in _worker
    work_item.run()
    │         └ <function _WorkItem.run at 0x7fee75fabd30>
    └ <concurrent.futures.thread._WorkItem object at 0x7fee7694f970>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/concurrent/futures/thread.py", line 52, in run
    result = self.fn(*self.args, **self.kwargs)
             │    │   │    │       │    └ {}
             │    │   │    │       └ <concurrent.futures.thread._WorkItem object at 0x7fee7694f970>
             │    │   │    └ (<grpc._cython.cygrpc.RequestCallEvent object at 0x7fee769419a0>, <grpc._server._RPCState object at 0x7fee769427f0>, <functio...
             │    │   └ <concurrent.futures.thread._WorkItem object at 0x7fee7694f970>
             │    └ <function _unary_response_in_pool at 0x7fee7692bd30>
             └ <concurrent.futures.thread._WorkItem object at 0x7fee7694f970>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/grpc/_server.py", line 560, in _unary_response_in_pool
    response, proceed = _call_behavior(rpc_event, state, behavior,
                        │              │          │      └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation at 0x7fee76946ca0>
                        │              │          └ <grpc._server._RPCState object at 0x7fee769427f0>
                        │              └ <grpc._cython.cygrpc.RequestCallEvent object at 0x7fee769419a0>
                        └ <function _call_behavior at 0x7fee7692b940>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/grpc/_server.py", line 443, in _call_behavior
    response_or_iterator = behavior(argument, context)
                           │        │         └ <grpc._server._Context object at 0x7fee7694fa00>
                           │        └ user_id: 1
                           │          name: "\346\256\265\350\266\205123"
                           │          mobile: "18612917508"
                           │          address: "\345\215\216\344\270\2551code"
                           │          
                           └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation at 0x7fee76946ca0>

  File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/grpcext/_interceptor.py", line 272, in adaptation
    return self._interceptor.intercept_unary(
           │    │            └ <function OpenTracingServerInterceptor.intercept_unary at 0x7fee75fbd9d0>
           │    └ <grpc_opentracing._server.OpenTracingServerInterceptor object at 0x7fee75f9df40>
           └ <grpc_opentracing.grpcext._interceptor._InterceptorRpcMethodHandler object at 0x7fee769427c0>

  File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/_server.py", line 154, in intercept_unary
    response = handler(request, servicer_context)
               │       │        └ <grpc_opentracing._server._OpenTracingServicerContext object at 0x7fee7694fc70>
               │       └ user_id: 1
               │         name: "\346\256\265\350\266\205123"
               │         mobile: "18612917508"
               │         address: "\345\215\216\344\270\2551code"
               │         
               └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation.<locals>.handler at 0x7fee76946940>

> File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/grpcext/_interceptor.py", line 269, in handler
    return self._rpc_method_handler.unary_unary(request,
           │    │                   │           └ user_id: 1
           │    │                   │             name: "\346\256\265\350\266\205123"
           │    │                   │             mobile: "18612917508"
           │    │                   │             address: "\345\215\216\344\270\2551code"
           │    │                   │             
           │    │                   └ _tuplegetter(4, 'Alias for field number 4')
           │    └ RpcMethodHandler(request_streaming=False, response_streaming=False, request_deserializer=<built-in method FromString of Gener...
           └ <grpc_opentracing.grpcext._interceptor._InterceptorRpcMethodHandler object at 0x7fee769427c0>

  File "/Users/www/learn-note/gpmall/daoServer/order/handler/order.py", line 156, in CreateOrder
    ret = p.send_message_in_transaction(msg, self.local_execute, user_args=None)
          │ │                           │    │    └ <function OrderServicer.local_execute at 0x7fee75c48940>
          │ │                           │    └ <handler.order.OrderServicer object at 0x7fee76921ac0>
          │ │                           └ <rocketmq.client.Message object at 0x7fee7694fd60>
          │ └ <function TransactionMQProducer.send_message_in_transaction at 0x7fee7544fa60>
          └ <rocketmq.client.TransactionMQProducer object at 0x7fee7694fcd0>

  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/rocketmq/client.py", line 335, in send_message_in_transaction
    ffi_check(
    └ <function ffi_check at 0x7fee754379d0>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/rocketmq/exceptions.py", line 44, in ffi_check
    raise exc_cls(msg)
          │       └ 'msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> li...
          └ <class 'rocketmq.exceptions.RocketMQException'>

rocketmq.exceptions.RocketMQException: msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53
2022-08-16 02:20:32.956 | ERROR    | grpc_opentracing.grpcext._interceptor:handler:269 - An error has been caught in function 'handler', process 'MainProcess' (29350), thread 'ThreadPoolExecutor-0_0' (123145521827840):
Traceback (most recent call last):

  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 930, in _bootstrap
    self._bootstrap_inner()
    │    └ <function Thread._bootstrap_inner at 0x7fee736e9c10>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145521827840)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 973, in _bootstrap_inner
    self.run()
    │    └ <function Thread.run at 0x7fee736e9940>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145521827840)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 910, in run
    self._target(*self._args, **self._kwargs)
    │    │        │    │        │    └ {}
    │    │        │    │        └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145521827840)>
    │    │        │    └ (<weakref at 0x7fee76941e50; to 'ThreadPoolExecutor' at 0x7fee75f9dfa0>, <_queue.SimpleQueue object at 0x7fee75fb7450>, None,...
    │    │        └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145521827840)>
    │    └ <function _worker at 0x7fee75fab8b0>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 123145521827840)>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/concurrent/futures/thread.py", line 77, in _worker
    work_item.run()
    │         └ <function _WorkItem.run at 0x7fee75fabd30>
    └ <concurrent.futures.thread._WorkItem object at 0x7fee7694f970>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/concurrent/futures/thread.py", line 52, in run
    result = self.fn(*self.args, **self.kwargs)
             │    │   │    │       │    └ {}
             │    │   │    │       └ <concurrent.futures.thread._WorkItem object at 0x7fee7694f970>
             │    │   │    └ (<grpc._cython.cygrpc.RequestCallEvent object at 0x7fee769419a0>, <grpc._server._RPCState object at 0x7fee769427f0>, <functio...
             │    │   └ <concurrent.futures.thread._WorkItem object at 0x7fee7694f970>
             │    └ <function _unary_response_in_pool at 0x7fee7692bd30>
             └ <concurrent.futures.thread._WorkItem object at 0x7fee7694f970>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/grpc/_server.py", line 560, in _unary_response_in_pool
    response, proceed = _call_behavior(rpc_event, state, behavior,
                        │              │          │      └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation at 0x7fee76946ca0>
                        │              │          └ <grpc._server._RPCState object at 0x7fee769427f0>
                        │              └ <grpc._cython.cygrpc.RequestCallEvent object at 0x7fee769419a0>
                        └ <function _call_behavior at 0x7fee7692b940>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/grpc/_server.py", line 443, in _call_behavior
    response_or_iterator = behavior(argument, context)
                           │        │         └ <grpc._server._Context object at 0x7fee7694fa00>
                           │        └ user_id: 1
                           │          name: "\346\256\265\350\266\205123"
                           │          mobile: "18612917508"
                           │          address: "\345\215\216\344\270\2551code"
                           │          
                           └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation at 0x7fee76946ca0>

  File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/grpcext/_interceptor.py", line 272, in adaptation
    return self._interceptor.intercept_unary(
           │    │            └ <function OpenTracingServerInterceptor.intercept_unary at 0x7fee75fbd9d0>
           │    └ <grpc_opentracing._server.OpenTracingServerInterceptor object at 0x7fee75f9df40>
           └ <grpc_opentracing.grpcext._interceptor._InterceptorRpcMethodHandler object at 0x7fee769427c0>

  File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/_server.py", line 154, in intercept_unary
    response = handler(request, servicer_context)
               │       │        └ <grpc_opentracing._server._OpenTracingServicerContext object at 0x7fee7694fc70>
               │       └ user_id: 1
               │         name: "\346\256\265\350\266\205123"
               │         mobile: "18612917508"
               │         address: "\345\215\216\344\270\2551code"
               │         
               └ <function _InterceptorRpcMethodHandler.unary_unary.<locals>.adaptation.<locals>.handler at 0x7fee76946940>

> File "/Users/www/learn-note/gpmall/daoServer/common/grpc_opentracing/grpcext/_interceptor.py", line 269, in handler
    return self._rpc_method_handler.unary_unary(request,
           │    │                   │           └ user_id: 1
           │    │                   │             name: "\346\256\265\350\266\205123"
           │    │                   │             mobile: "18612917508"
           │    │                   │             address: "\345\215\216\344\270\2551code"
           │    │                   │             
           │    │                   └ _tuplegetter(4, 'Alias for field number 4')
           │    └ RpcMethodHandler(request_streaming=False, response_streaming=False, request_deserializer=<built-in method FromString of Gener...
           └ <grpc_opentracing.grpcext._interceptor._InterceptorRpcMethodHandler object at 0x7fee769427c0>

  File "/Users/www/learn-note/gpmall/daoServer/order/handler/order.py", line 156, in CreateOrder
    ret = p.send_message_in_transaction(msg, self.local_execute, user_args=None)
          │ │                           │    │    └ <function OrderServicer.local_execute at 0x7fee75c48940>
          │ │                           │    └ <handler.order.OrderServicer object at 0x7fee76921ac0>
          │ │                           └ <rocketmq.client.Message object at 0x7fee7694fd60>
          │ └ <function TransactionMQProducer.send_message_in_transaction at 0x7fee7544fa60>
          └ <rocketmq.client.TransactionMQProducer object at 0x7fee7694fcd0>

  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/rocketmq/client.py", line 335, in send_message_in_transaction
    ffi_check(
    └ <function ffi_check at 0x7fee754379d0>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/rocketmq/exceptions.py", line 44, in ffi_check
    raise exc_cls(msg)
          │       └ 'msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> li...
          └ <class 'rocketmq.exceptions.RocketMQException'>

rocketmq.exceptions.RocketMQException: msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53
2022-08-16 02:23:30.309 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:23:30.309 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:23:30.309 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:23:36.785 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:23:36.785 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:23:36.785 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:26:37.247 | INFO     | handler.order:CreateOrder:178 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:26:37.247 | INFO     | handler.order:CreateOrder:178 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:26:37.247 | INFO     | handler.order:CreateOrder:178 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:27:23.207 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:27:23.207 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:27:23.207 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:27:24.904 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:27:24.904 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:27:24.904 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:27:27.213 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:27:27.213 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:27:27.213 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:27:52.743 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:27:52.743 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:27:52.743 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:28:52.783 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:28:52.783 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:28:52.783 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:28:56.142 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:28:56.142 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:28:56.142 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:29:32.081 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:29:32.081 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:29:32.081 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:32:04.867 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:32:04.867 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:32:04.867 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:32:07.480 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:32:07.480 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:32:07.480 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:32:40.093 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:32:40.093 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:32:40.093 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:32:46.910 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:32:46.910 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:32:46.910 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:34:40.696 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:34:40.696 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:34:40.696 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:36:29.574 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:36:29.574 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:36:29.574 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:36:40.304 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:36:40.304 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:36:40.304 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:37:07.194 | INFO     | handler.order:CreateOrder:161 - ('The producer group[transProducerGroup] has been created before, specify another name please.,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp line:79',)
2022-08-16 02:37:07.194 | INFO     | handler.order:CreateOrder:161 - ('The producer group[transProducerGroup] has been created before, specify another name please.,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp line:79',)
2022-08-16 02:37:07.194 | INFO     | handler.order:CreateOrder:161 - ('The producer group[transProducerGroup] has been created before, specify another name please.,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp line:79',)
2022-08-16 02:38:22.480 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:38:22.480 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:38:22.480 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:38:27.514 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:38:27.514 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:38:27.514 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:38:30.334 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:38:30.334 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:38:30.334 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:38:35.728 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:38:35.728 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:38:35.728 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:46:04.752 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:46:04.752 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:46:04.752 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:46:07.841 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:46:07.841 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:46:07.841 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:46:18.176 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:46:18.176 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:46:18.176 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:52:37.770 | INFO     | __main__:run:70 - service start...: 192.168.8.222:4347
2022-08-16 02:52:37.770 | INFO     | __main__:run:70 - service start...: 192.168.8.222:4347
2022-08-16 02:52:37.770 | INFO     | __main__:run:70 - service start...: 192.168.8.222:4347
2022-08-16 02:53:01.396 | INFO     | __main__:f:115 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:53:01.396 | INFO     | __main__:f:115 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:53:01.396 | INFO     | __main__:f:115 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:54:27.694 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:54:27.694 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:54:27.694 | INFO     | __main__:run:71 - service start...: 192.168.8.222:4347
2022-08-16 02:55:16.514 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:55:16.514 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:55:16.514 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 02:55:19.910 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:55:19.910 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:55:19.910 | INFO     | __main__:f:116 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:56:04.991 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 02:56:04.991 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 02:56:04.991 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 02:56:28.125 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:56:28.125 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 02:56:28.125 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 03:13:57.635 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 03:13:57.635 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 03:13:57.635 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 03:15:00.380 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 03:15:00.380 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 03:15:00.380 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 03:15:01.898 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 03:15:01.898 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 03:15:01.898 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 03:15:18.924 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 03:15:18.924 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 03:15:18.924 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 03:15:20.736 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 03:15:20.736 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 03:15:20.736 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 03:15:40.500 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 03:15:40.500 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 03:15:40.500 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 03:15:42.009 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 03:15:42.009 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 03:15:42.009 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 03:26:59.357 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 03:26:59.357 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 03:26:59.357 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 03:31:39.243 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 03:31:39.243 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 03:31:39.243 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 03:31:46.227 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 03:31:46.227 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 03:31:46.227 | INFO     | handler.order:CreateOrder:161 - ('msg: No route info of this topic: transTopic,error:-1,in file rocketmq-client-cpp/src/producer/DefaultMQProducerImpl.cpp> line:418,error:-1,in file </Users/shannon/work/OpenSource/Apache/rocketmq-client-cpp/src/producer/TransactionMQProducerImpl.cpp line:53',)
2022-08-16 03:31:51.951 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 03:31:51.951 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 03:31:51.951 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 03:42:56.042 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 03:42:56.042 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 03:42:56.042 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 03:49:18.584 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 03:49:18.584 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 03:49:18.584 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 04:56:13.587 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 04:56:13.587 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 04:56:13.587 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 04:56:13.643 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:13.643 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:13.643 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:14.666 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:14.666 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:14.666 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:15.679 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:15.679 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:15.679 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:44.627 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:44.627 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:44.627 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:45.633 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:45.633 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:45.633 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:46.640 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:46.640 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:46.640 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:47.647 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:47.647 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:47.647 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:48.654 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:48.654 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:48.654 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:49.659 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:49.659 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:49.659 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:50.667 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:50.667 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:50.667 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:51.671 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:51.671 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:51.671 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:52.678 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:52.678 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:52.678 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:53.685 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:53.685 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:53.685 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:54.690 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:54.690 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:54.690 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:55.695 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:55.695 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:56:55.695 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:57:35.722 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:57:35.722 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:57:35.722 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:57:41.962 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 04:57:41.962 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 04:57:41.962 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 04:57:48.476 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 04:57:48.476 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 04:57:48.476 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 04:58:35.750 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:58:35.750 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 04:58:35.750 | INFO     | handler.order:cancel_timeout_order:79 - 发警告短信给开发人员 
2022-08-16 05:00:11.327 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 05:00:11.327 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 05:00:11.327 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 05:00:13.883 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 05:00:13.883 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 05:00:13.883 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 05:00:35.757 | INFO     | handler.order:cancel_timeout_order:78 - 发警告短信给开发人员 
2022-08-16 05:00:35.757 | INFO     | handler.order:cancel_timeout_order:78 - 发警告短信给开发人员 
2022-08-16 05:00:35.757 | INFO     | handler.order:cancel_timeout_order:78 - 发警告短信给开发人员 
2022-08-16 05:03:45.808 | INFO     | handler.order:cancel_timeout_order:78 - 发警告短信给开发人员 
2022-08-16 05:03:45.808 | INFO     | handler.order:cancel_timeout_order:78 - 发警告短信给开发人员 
2022-08-16 05:03:45.808 | INFO     | handler.order:cancel_timeout_order:78 - 发警告短信给开发人员 
2022-08-16 05:16:34.650 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 05:16:34.650 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 05:16:34.650 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 05:16:36.065 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 05:16:36.065 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 05:16:36.065 | INFO     | __main__:run:69 - service start...: 192.168.8.222:4347
2022-08-16 05:48:57.830 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 05:48:57.830 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
2022-08-16 05:48:57.830 | INFO     | __main__:f:114 - 退出订单服务 service stop...:192.168.8.222:4347
